sdk/examples/poet_python/Dockerfile-installed

```diff
@@ -0,0 +1,227 @@
+# Copyright 2018 Cargill Incorporated
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# docker build -f sdk/examples/poet_python/Dockerfile-installed -t sawtooth-poet-engine-installed .
+
+# -------------=== signing build ===-------------
```

Comment:
Can you please include the usual proxy environment variables?

Reply:
sure

Reply:
Can you clarify what these need to be? I don't see anything in the other PoET dockerfiles.